app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is defined as a class attribute, which means it is shared across all instances of the `Animal` class. This can lead to unexpected behavior if multiple instances are created and modified. Consider making `alive` an instance attribute or managing it differently.

Reply:
I has to be like this so we have the list of animals